packages/cli/patches/@coveord+platform-client+18.15.2.patch

```diff
@@ -0,0 +1,54 @@
+diff --git a/node_modules/@coveord/platform-client/dist/definitions/resources/ResourceSnapshots/ResourceSnapshots.d.ts b/node_modules/@coveord/platform-client/dist/definitions/resources/ResourceSnapshots/ResourceSnapshots.d.ts
```

Comment:
I'm really not in favor of doing a patch on dependencies where we have a bit of control (I mean, it's in the cie)
I can't approve this PR with this.

Reply:
Blocked by https://github.com/coveo/cli/pull/240/

Reply:
Not blocked anymore